.travis.yml

```diff
@@ -48,7 +48,9 @@ script:
   - >-
     kubecfg validate -v kubeapps.jsonnet || :
   - ./kubeapps.sh up
-  - ./tests.sh $(./kubeapps.sh dashboard --url)
+  # We can't port-forward in Travis, so instead we patch the service type to node port
```

Comment:
I'm happy with merging this for now, but can you leave a followup issue (assigned to me) to install the extra bits we need for port-forward to work on travis too.